### DISCUSSION TITLE
Admin can edit comments without a track.

### DISCUSSION THREAD

**Discussion by sandeepy1** [requarks/wiki#5918] [Help / Questions] [answered]
I am exploring Wiki.js from last week, and noticed that admin can edit other user's comments without leaving a track. Admin can misuse this power if he/she wants to. It can be seen as a limitation. Do you have any plan to change this in future?

**Comment by fnafox** [comment]
If an admin had malicious intent they could just as well make changes in the database even if they couldn't directly from the app. Still worth looking into imo.

**Comment by NGPixel** [comment, marked as answer]
Auditing is coming in v3 _(which will include edited comments)_.

However, if an admin is doing malicious editing, you may have bigger issues...
